### PR TITLE
fix: separate after event organizers & speaker mails

### LINKED ISSUE #6620: After-Event Mail: Separate Email Template of Speaker and Owner/Organizer
There is an after-event message, that should not go to the speaker and organizer at the same time, which can be viewed at https://eventyay.com/admin/messages/list .

Screenshot from 2019-09-16 10-59-01

Instead there should be
a) a message to the owner and organizers
b) a message to the speaker
These message should be differently and don't include upcoming events.

1. Please implement a message to the "Owner, Organizer"
Subject: {eventname} is over. Thank you for using Eventyay!
Email: Hi {email}, you event {eventname] is over. Thank you for using the Eventyay platform for your event. We hope you enjoyed it. 

2. Implement a message only for accepted speakers
Subject: {eventname} is over. Thank you for participating!
Email: Hi {email},thank you for participating in our {eventname]. We hope you enjoyed it and keep in touch.

### PR DESCRIPTION
**Pull request by progmatic-99**
Fixes #6620

### CODE REVIEW

File: app/api/helpers/mail.py [hunk 1]
```diff
@@ -725,3 +725,45 @@ def send_email_to_moderator(video_stream_moderator):
             settings=get_settings(),
         ),
     )
+
+
+def send_email_after_event(email, event):
```
Comment (maze-runnar) [inline app/api/helpers/mail.py:730]: you have to add the functions in schedule jobs too. please test PR completely. If you have not setUp email thing. Just use logger and see if it's working.

**Review by maze-runnar — CHANGES_REQUESTED** [review]

File: app/api/helpers/mail.py (hunk 1 above)
Comment (progmatic-99) [reply app/api/helpers/mail.py:730]: I tried testing using logger, but there is no log regarding the after event mail. How do I set up the SendGrid ?
I got the API key from SendGrid. Do I need to create a sender ID too??

For the logger, I just have to set up celery and redis??

**Comment by progmatic-99** [comment]
@maze-runnar Please test this.

**Comment by maze-runnar** [comment]
> @maze-runnar Please test this.

Make the changes, we discussed yesterday.

**Review by maze-runnar — APPROVED** [review]

**Review by mariobehling — APPROVED** [review]

**Pull request merged**